src/fmu/sumo/uploader/caseondisk.py

```diff
@@ -191,14 +191,32 @@ def register(self):
             sumo_parent_id (uuid4): Unique ID for this case on Sumo
         """
 
-        logger.info("Registering case on Sumo")
-
-        sumo_parent_id = self._upload_case_metadata(self.case_metadata)
-        self._sumo_parent_id = sumo_parent_id
-
-        logger.info("Case registered. SumoID: {}".format(sumo_parent_id))
-
-        return sumo_parent_id
+        logger.info("About to register case on Sumo")
+
+        try: 
+            sumo_parent_id = self._upload_case_metadata(self.case_metadata)
+            self._sumo_parent_id = sumo_parent_id
+
+            logger.info("Case registered. SumoID: {}".format(sumo_parent_id))
+
+            return sumo_parent_id
+        except Exception as err:
+            print("\n\033[31m"
+                "Error during registering case on Sumo. "
+                "\nFile uploads will also fail. "
+                "\033[0m")
+            if '401 Unauthorized' in str(err):
+                print("\033[31m"
+                    "Did you login to Sumo?"
```

Comment:
Instead of "Did you login to Sumo?", suggest "Please verify that you are logged in to Sumo, by running `sumo_login` in a Unix terminal window."